eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -105,332 +105,396 @@ groups:
           central: +0.1959500850585235
           min:     +0.1959500850585235
           max:     +0.1959500850585235
+          latex:    '$\alpha_{+,0}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^f+_1@BSZ2015' :
           central: -0.7763565146997043
           min:     -0.7763565146997043
           max:     -0.7763565146997043
+          latex:    '$\alpha_{+,1}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^f+_2@BSZ2015' :
           central: -0.19660537219243024
           min:     -0.19660537219243024
           max:     -0.19660537219243024
+          latex:    '$\alpha_{+,2}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^f0_1@BSZ2015' :
           central: -0.01446803146697799
           min:     -0.01446803146697799
           max:     -0.01446803146697799
+          latex:    '$\alpha_{0,1}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^f0_2@BSZ2015' :
           central: -0.19529667411047308
           min:     -0.19529667411047308
           max:     -0.19529667411047308
+          latex:    '$\alpha_{0,2}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^fT_0@BSZ2015' :
           central: +0.17311654822482725
           min:     +0.17311654822482725
           max:     +0.17311654822482725
+          latex:    '$\alpha_{T,0}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^fT_1@BSZ2015' :
           central: -0.40903716524130396
           min:     -0.40903716524130396
           max:     -0.40903716524130396
+          latex:    '$\alpha_{T,1}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       'B->pi::alpha^fT_2@BSZ2015' :
           central: -0.2580355775883392
           min:     -0.2580355775883392
           max:     -0.2580355775883392
+          latex:    '$\alpha_{T,2}^{B \to \pi,\text{BSZ2015}}$'
           comments: ''
       # form factor parameters for B->K for [BCL2008] parametrization
       'B->K::f_+(0)@BCL2008' :
           central: +0.0
           min:     +0.0
           max:     +0.0
+          latex:    '$f_+(0)^{B\to K}$'
```

Comment:
`$f_+^{B\to K}(0)$` would be more compact, no?

Reply:
In concur.